packages/manganis/manganis-macro/src/asset.rs

```diff
@@ -19,19 +25,28 @@ fn resolve_path(raw: &str) -> Result<PathBuf, AssetParseError> {
     // /users/dioxus/dev/app/assets/blah.css
     let manifest_dir = std::env::var("CARGO_MANIFEST_DIR")
         .map(PathBuf::from)
+        .map(std::path::absolute)
+        .unwrap()
         .unwrap();
 
     // 1. the input file should be a pathbuf
     let input = PathBuf::from(raw);
 
     // 2. absolute path to the asset
-    manifest_dir
-        .join(raw.trim_start_matches('/'))
-        .canonicalize()
-        .map_err(|err| AssetParseError::AssetDoesntExist {
-            err,
-            path: input.clone(),
-        })
+    let path =
+        std::path::absolute(manifest_dir.join(raw.trim_start_matches('/'))).map_err(|err| {
```

Comment:
`std::path::absolute` does not check if the path exists (unlike canonicalize) so this error does not trigger when the path doesn't exist